src/features/groups/components/GroupCardList/UserCount.tsx

```diff
@@ -0,0 +1,15 @@
+import { PersonCircleIcon } from '@/assets/images/icons';
+import { Icon } from '@/components/ui';
+
+export const UserCount = ({ userCount }: { userCount: number }) => {
+  return (
+    <div className="flex items-center gap-1.5">
+      <div className="relative w-16 h-8">
+        <Icon icon={PersonCircleIcon} size={6} color="gray" className="absolute left-0 top-1" />
+        <Icon icon={PersonCircleIcon} size={6} color="gray" className="absolute left-5 top-1" />
+        <Icon icon={PersonCircleIcon} size={6} color="gray" className="absolute left-10 top-1" />
+      </div>
```

Comment:
Although the icons look center-aligned with the count number, aligning them using `top-1` is not an appropriate approach.

Instead, you could use `flex items-center` to achieve the same thing:
```tsx
      <div className="relative w-16 h-8 flex items-center">
        <Icon icon={PersonCircleIcon} size={6} color="gray" className="absolute left-0" />
        <Icon icon={PersonCircleIcon} size={6} color="gray" className="absolute left-5" />
        <Icon icon={PersonCircleIcon} size={6} color="gray" className="absolute left-10" />
      </div>
```